src/sockets/folder.socket.ts

```diff
@@ -0,0 +1,26 @@
+import * as socketIo from 'socket.io';
```

Comment:
rename the files to be like `socket.folder.ts `

Reply:
renamed that and created only one generic class called in the socket file